odoo/models.py

```diff
@@ -3458,6 +3458,8 @@ def _write(self, vals):
                     val = field.convert_to_column(vals[name], self, vals)
                     self.env['ir.translation']._set_ids(
                         tname, 'model', self.env.lang, self.ids, val, src_trans)
+                    self.env['ir.translation']._set_source(
+                        tname, 'model', self.ids, src_trans)
```

Comment:
Wouldn't it make sense to update _set_ids to handle the _set_source use case instead? Also possibly rename it because _set_ids is an awful name for it.

Reply:
The main difference is that `_set_ids`  (as the name obviously indicated) is only working on one record but it could be changed I guess (just check `l10n_multilang` that calls this method too.

Reply:
But as far as I can see this is the only call to _set_source and it immediately follows a _set_ids call, so there's effectively no difference?